pkg/command/context.go

```diff
@@ -168,7 +178,7 @@ func createNewContext() (context *configtypes.Context, err error) {
 	err = component.Prompt(
 		&component.PromptConfig{
 			Message: "Select context creation type",
-			Options: []string{"Control plane endpoint", "Local kubeconfig"},
+			Options: []string{"Control plane endpoint", "Local kubeconfig", "self-managed TMC"},
```

Comment:
```suggestion
			Options: []string{"Control plane endpoint", "Local kubeconfig", "Self-managed TMC"},
```
To be consistent with the other two